### ISSUE TITLE
Popup dialog disappears when editing image with a caption

### ISSUE THREAD

**Issue by avewells** [froala/wysiwyg-editor#4054]
##### Expected behavior.
After adding a caption to an image I should be able to see the popup dialog to further modify the image with commands that require `fr-popup`.

##### Actual behavior.
After adding a caption to an image the `fr-popup` does not get the correct `top` and `left` properties so the dialog box goes to the edges of the screen making it hidden/hard to find.

##### Steps to reproduce the problem.
1. Open iFrame example at https://froala.com/wysiwyg-editor/examples/iframe/
2. Insert an image
3. Add a caption to the image
4. You should notice that the dialog "disappears" to the top of the screen

##### Editor version.
- 3.1.1
- Whatever is used on the demo

##### OS.
- Mac Catalina 10.15.4

##### Browser.
- Chrome 85

##### Recording.
![2020-09-22 15 41 33](https://user-images.githubusercontent.com/5642174/93935460-4649a600-fcea-11ea-9f66-999228731234.gif)

**Comment by Subha** [comment]
Any updates on this item?

**Comment by avewells** [comment]
@Subha  Still an issue according to my testing.

**Comment by Subha** [comment]
> @Subha Still an issue according to my testing.

yup, i was wondering if there is any update from the Froala team

**Comment by sergsagan** [comment]
I have the same problem

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
